rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -1353,6 +1385,76 @@ public ResponseEntity loadAssessmentSummaryInfo(
         return new ResponseEntity<>(assessmentSummaryMap, HttpStatus.OK);
     }
 
+    @Operation(
+            summary = "Check cyclic hierarchy of a release with other releases.",
+            description = "Check cyclic hierarchy of a release with other releases."
+    )
```

Comment:
Please add the `Releases` tag to group endpoint correctly.
```suggestion
    @Operation(
            summary = "Check cyclic hierarchy of a release with other releases.",
            description = "Check cyclic hierarchy of a release with other releases.",
            tags = {"Releases"}
    )
```